lib/matplotlib/widgets.py

```diff
@@ -220,7 +220,7 @@ def _motion(self, event):
         if self.ignore(event):
             return
         c = self.hovercolor if event.inaxes == self.ax else self.color
-        if c != self.ax.get_facecolor():
+        if colors.to_rgba(c) != colors.to_rgba(self.ax.get_facecolor()):
```

Comment:
You can use `same_color` here:
```suggestion
        if not colors.same_color(c, self.ax.get_facecolor()):
```

Reply:
Thanks for pointing that out, much cleaner that way